src/app/pages/my-request-detail/my-request-detail.component.ts

```diff
@@ -112,6 +114,11 @@ export class MyRequestDetailComponent implements OnInit {
 
   selectAllElements = false;
 
+  /**
+   * Signal to attach to subscriptions to trigger when they should be unsubscribed.
+   */
+  private unsubscribe$ = new Subject<void>();
```

Comment:
Do we need new tests on this component to test this works as expected (which it seems it does) but mostly to ensure we don't break anything in the future?

Reply:
Started but abandoned to allow us to close down some in-flight branches on this page